inception-kb/src/main/java/de/tudarmstadt/ukp/inception/kb/IriConstants.java

```diff
@@ -21,20 +21,36 @@
 import java.util.ArrayList;
 import java.util.Arrays;
 import java.util.Collections;
+import java.util.HashSet;
 import java.util.List;
+import java.util.Set;
 
 import org.eclipse.rdf4j.model.IRI;
 import org.eclipse.rdf4j.model.ValueFactory;
 import org.eclipse.rdf4j.model.impl.SimpleValueFactory;
 import org.eclipse.rdf4j.model.vocabulary.OWL;
 import org.eclipse.rdf4j.model.vocabulary.RDF;
 import org.eclipse.rdf4j.model.vocabulary.RDFS;
+import org.eclipse.rdf4j.model.vocabulary.XMLSchema;
 
 public class IriConstants
 {
 
+    public static final String INCEPTION_SCHEMA_NAMESPACE = "http://www.ukp.informatik.tu-darmstadt.de/inception/schema-1.0#";
+
+    public static final String INCEPTION_NAMESPACE = "http://www.ukp.informatik.tu-darmstadt.de/inception/1.0#";
+
     public static final String WIKIDATA_NAMESPACE = "https://www.wikidata.org/wiki/";
 
+    public static final String[] IMPLICIT_NAMESPACES = { RDF.NAMESPACE, RDFS.NAMESPACE,
```

Comment:
To make it simple, I would either use Set or array for a collection of IRI, but not both next to each other.